documentation/modules/security/con-securing-kafka-authentication.adoc

```diff
@@ -95,6 +95,10 @@ mTLS listener configuration in the `Kafka` resource requires the following:
 * `tls: true` to specify TLS encryption and server authentication
 * `authentication.type: TLS` to specify the client authentication
 
+When a Kafka cluster is created by the Cluster Operator, it creates a new secret with the name `<cluster_name>-cluster-ca-cert`.
+The secret contains a private and public key for mTLS.
```

Comment:
I know it's not related to this PR but I am a little bit picky about the usage of "mTLS" here.
I mean ... the private and public key in this Secret is about enabling encryption and server authentication only.
In order to have "mTLS" so adding client authentication, you need the Secret on the other "side" with public and private keys for client.
So I was wondering if saying " ... for mTLS" is misleading here.

Reply:
Yes. I don't want to repeat what we have elsewhere, but I've expanded a little to show the _bigger picture_

When a Kafka cluster is created by the Cluster Operator, it creates a new secret with the name `<cluster_name>-cluster-ca-cert`.
The secret contains a public CA certificate.
The public CA certificate is in xref:certificates-and-secrets-formats-{context}[PEM and PKCS #12 format].
To verify a Kafka cluster, you add the public CA certificate to the truststore in your client configuration.
To verify the client, you add user credentials to keystore in your client configuration.
For more information on configuring a client for mTLS, see xref:[con-securing-client-authentication-str].

Reply:
Expansion sounds better to me but ...

> To verify the client, you add user credentials to keystore in your client configuration.

What do you mean by "credentials" here?

Reply:
Is this enough:

_To verify a client, you add a user certificate and key to the keystore in your client configuration._

Reply:
Yeah, much better thanks!